rpcd/playbooks/roles/rpc_maas/tasks/local.yml

```diff
@@ -195,6 +195,21 @@
 
 - include: local_setup.yml
   vars:
+    check_name: neutron_metadata_agent_proxy_check
+    check_details: file=neutron_metadata_local_check.py,args={{ internal_vip_address }}
+    check_period: "{{ maas_check_period }}"
+    check_timeout: "{{ maas_check_timeout }}"
+    alarms:
+      - { 'name': 'neutron_metadata_agent_proxy_status', 'criteria': ':set consecutiveCount={{ maas_alarm_local_consecutive_count }} if (metric["neutron-metadata-agent-proxy_status"] != 1) { return new AlarmStatus(CRITICAL, "neutron-metadata-agent-proxy non-responsive"); }' }
```

Comment:
For now the check only runs on the first container it finds, so I did not go so far as to support duplicate containers on a host yet. This doesn't seem necessary so pulling back a bit.